cmd/openapi/transform.go

```diff
@@ -76,6 +77,13 @@ var cleanupCmd = &model.ExecutableCommand[basicFlagsI]{
 	Flags: basicFlags,
 }
 
+var formatCmd = &model.ExecutableCommand[basicFlagsI]{
+	Usage: "format",
+	Short: "Format a given OpenAPI document",
```

Comment:
Might be worth expanding on what exactly "format" means

Reply:
Updated this section